e2e/cypress/integration/translations/autoTranslations.ts

```diff
@@ -0,0 +1,94 @@
+import { deleteProject } from '../../common/apiCalls';
+import { ProjectDTO } from '../../../../webapp/src/service/response.types';
+import {
+  create4Translations,
+  createTranslation,
+  selectLangsInLocalstorage,
+  translationsBeforeEach,
+  visitTranslations,
+} from '../../common/translations';
+import { waitForGlobalLoading } from '../../common/loading';
+import { HOST } from '../../common/constants';
+import { assertHasState } from '../../common/state';
+
+describe('Translation memory', () => {
+  let project: ProjectDTO = null;
+
+  beforeEach(() => {
+    translationsBeforeEach()
+      .then((p) => (project = p))
+      .then(() => {
+        create4Translations(project.id);
+        selectLangsInLocalstorage(project.id, ['en', 'cs']);
+      });
+  });
+
+  afterEach(() => {
+    deleteProject(project.id);
+  });
+
+  it("won't trigger auto translation when not enabled", () => {
+    visit();
+    waitForGlobalLoading();
+    createTranslation('mykey', 'Cool translated text 1');
+    waitForGlobalLoading();
+    cy.gcy('translations-table-cell').contains('mykey').should('be.visible');
+    cy.gcy('translations-table-cell')
+      .filter(':contains("Cool translated text 1")')
+      .should('have.length', 2);
+    cy.gcy('translations-table-cell')
+      .filter(':contains("Studený přeložený text 1")')
+      .should('have.length', 1);
+  });
+
+  it('will translate with machine translations', () => {
```

Comment:
Don't fix it now, but please don't name the tests in the future tense, it was my bulls**t, sorry for that. I'll fix it someday in the current tests.

Reply:
👍